setup.cfg

```diff
@@ -63,7 +63,13 @@ include = torchgeo*
 # Optional dataset requirements
 datasets =
     h5py
+    # loading .las point clouds (idtrees) laspy 2+ required for Python 3.6+ support
+    laspy>=2.0.0
+    # open3d will add add support for python 3.9 in pypi in v0.14. v0.11.2 last version for tests to pass
+    open3d>=0.11.2;python_version<'3.9'
     opencv-python
+    # pandas > v1.0 for python 3.9 support
+    pandas>=1.0
```

Comment:
It sounds like older versions of pandas work fine

Reply:
pandas<v1.0 doesn't work with Python 3.9 on Ubuntu for me. Not sure why we would support a version that doesn't work for a Python version we support.

Reply:
The minimum version of a dependency is the oldest version that includes the features we need. It has nothing to do with which Python versions that dependency supports. Many users who are stuck with Python 3.6 are unable to build newer versions of dependencies or are stuck on systems where a system administrator is required to update the system installations of dependencies.

Reply:
Ahh ok well I'm just using read_csv and join so I guess hiw far do we go back then? Seems like that would be a very early feature of pandas.

Reply:
Python 3.6 support was added in pandas v0.19.1. Testing now